Automattic/wp-calypso
# Blocks: Remove GlobalNotice block

Remove a block only used to render one notice in one place.

We have existing practices for displaying notices, this block had some limitations and it was more work to improve the block than to eliminate it. This also encourages the same methodology to be used across the codebase.

**Beware**

I don't know where this notice would, in theory, appear in production. I was looking for it but have been unable to get the notice rendered.

## Testing
1. **I expected a notice but never saw one in production** Compare the notice "Creating your account…" displayed in production with the notice displayed on this branch when creating a new social user:
   - https://wordpress.com/start/account/?debug
   - https://calypso.live/start/account/?debug&branch=remove/global-notice-block
1. Compare the action log, they should be very similar although may not have exactly the same order:
   ```js
   actionLog.filter( /^NOTICE_/ )
   ```
1. Verify this action is only called during social account creation on the provided route
1. Verify the removed components, selectors and state were only used by the code removed in this PR

Closes #24568 (supersedes)
